Competitive Coding/Math/Catalan_Numbers/catalan_recursive.py

```diff
@@ -0,0 +1,10 @@
+def catalan_numbers(n):
+    if n <=1 :		#The result will be 1, if the function takes an argument that's equal to or less than 1
+        return 1
+    res = 0		#Result has been initialised to 0
+    for i in range(n):
+        res += catalan_numbers(i) * catalan_numbers(n-i-1)	#The recursive function call
```

Comment:
where is the coefficient in this case?

Reply:
@prateekiiest what coefficient are you referring to?

Reply:
The combination C factor